packages/trpc/server/routers/viewer/teams/list.handler.ts

```diff
@@ -23,9 +24,14 @@ export const listHandler = async ({ ctx }: ListOptions) => {
     orderBy: { role: "desc" },
   });
 
-  return memberships.map(({ team, ...membership }) => ({
-    role: membership.role,
-    accepted: membership.accepted,
-    ...team,
-  }));
+  return memberships
+    .filter((mmship) => {
+      const metadata = teamMetadataSchema.parse(mmship.team.metadata);
+      return !metadata?.isOrganization;
+    })
```

Comment:
Filtering out organization teams per se. Tried from prisma query but couldn't make it work:
```js
metadata: {
  path: ["isOrganization"],
  not: true,
},
```

Reply:
Good enough for now